### ISSUE TITLE
Display images rotated by aspect ratio

### ISSUE THREAD

**Issue by owenjm** [kollerlukas/Camera-Roll-Android-App#107]
Great software!  I was wondering if you might consider including an option to display photos automatically rotated by aspect ratio (such that all available screen real-estate is used for each photo, regardless of the orientation)?  A few other gallery apps have this feature, which I find very useful.

Many thanks,
Owen

**Comment by owenjm** [comment]
Well, I would certainly find it useful, and given that it's a common feature in other gallery apps I suspect I'm not the only one.  I have rotation turned off on my phone (I'm using my phone almost 100% in portrait mode, as I suspect most people do), and this feature makes things easier and faster.

If you think about it, if you only use the phone's rotation sensor, you'll still need to rotate the phone exactly the same way when viewing photos (if the photo's in landscape orientation, you'll rotate it to landscape, just as you would with this feature selected) but this way it's 100& reliable and instantaneous ... and doesn't require turning on the rotation sensor :)

Anyway, if it was possible to make it an option that would be much appreciated, but thanks for writing Camera Roll (and for open sourcing it!) either way.

Cheers,
Owen

**Comment by PhilippWendler** [comment]
I agree this is useful for people who have turned off auto-rotation of the phone (I find auto-rotation annoying, but I always need to enable it before viewing photos and disable it again afterwards).

**Comment by poisonborz** [comment]
This is the only thing missing for me to switch to this app... but maybe it shouldn't be autorotate based on the aspect ratio... images could be rotated via rotate gesture while zooming, just like old QuickPic gallery - this way any any image could be viewed rotated in any direction, based on obvious user input.
